packages/core/tests/lib/express.test.ts

```diff
@@ -89,14 +88,14 @@ describe('Express', () => {
       app = express();
     });
 
-    it('set default rateLimiter', async (done) => {
+    it('set default rateLimiter', async () => {
       const rateLimiter = getRateLimiter();
       app.use('/', rateLimiter, (_req, res) => res.status(200).send('Welcome'));
 
-      done();
+      expect(1).toEqual(1)
```

Comment:
I'm assuming this is intentional, but it looks a little odd so I'm flagging just so you can confirm.

Reply:
I didn't really recall what the test was supposed to do if I'm being honest. Upgraded jest complains now about using `async/await` & `done` together so just wanted to make sure something gets assessed. Agreed that it's definitely weird and should be looked at, just didn't want to touch existing tests